src/EditorFeatures/VisualBasicTest/CodeActions/ExtractMethod/ExtractMethodTests.vb

```diff
@@ -293,5 +293,88 @@ End Class
 </Text>.Value.Replace(vbLf, vbCrLf),
 compareTokens:=False)
         End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsExtractMethod), Test.Utilities.CompilerTrait(Test.Utilities.CompilerFeature.Tuples)>
+        Public Async Function TestNoTuples() As Task
+
+            Await TestAsync(
+<Text>Class Program
+    Sub Main(args As String())
+        [|Dim x = 1|]
+        M(x)
+    End Sub
+
+    Private Sub M(x As Integer)
+    End Sub
+End Class
+
+Namespace System
+    Structure ValueTuple(Of T1, T2)
+    End Structure
+End Namespace
+</Text>.Value.Replace(vbLf, vbCrLf),
+<Text>Class Program
+    Sub Main(args As String())
+        Dim x As Integer = {|Rename:NewMethod|}()
+        M(x)
+    End Sub
+
+    Private Shared Function NewMethod() As Integer
+        Return 1
+    End Function
+
+    Private Sub M(x As Integer)
+    End Sub
+End Class
+
+Namespace System
+    Structure ValueTuple(Of T1, T2)
+    End Structure
+End Namespace
+</Text>.Value.Replace(vbLf, vbCrLf),
+compareTokens:=False)
```

Comment:
compareTokens:=false is only for if you are testin formatting.  If you don't care about formatting, then just elide this argument entirely.